### ISSUE TITLE
Fails to generate valid code for GLibD

### ISSUE THREAD

**Issue by ximion** [gtkd-developers/gir-to-d#19]
Hi!
At the moment, the GLibD build at Debian fails after the gir-to-d update with
```
[1/327] ldc2 -Iglibd-2.0@sha -I. -I.. -Igenerated -I../generated -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -enable-color -O -g -release -wi -relocation-model=pic   -of 'glibd-2.0@sha/obj-x86_64-linux-gnu_generated_glib_c_functions.d.o' -c ../obj-x86_64-linux-gnu/generated/glib/c/functions.d
FAILED: glibd-2.0@sha/obj-x86_64-linux-gnu_generated_glib_c_functions.d.o 
ldc2 -Iglibd-2.0@sha -I. -I.. -Igenerated -I../generated -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -enable-color -O -g -release -wi -relocation-model=pic   -of 'glibd-2.0@sha/obj-x86_64-linux-gnu_generated_glib_c_functions.d.o' -c ../obj-x86_64-linux-gnu/generated/glib/c/functions.d
../obj-x86_64-linux-gnu/generated/glib/c/functions.d(1163): Error: found `const` when expecting `)`
../obj-x86_64-linux-gnu/generated/glib/c/functions.d(1163): Error: semicolon expected following function declaration
../obj-x86_64-linux-gnu/generated/glib/c/functions.d(1163): Error: declaration expected, not `*`
../obj-x86_64-linux-gnu/generated/glib/c/functions.d(1187): Error: found `const` when expecting `)`
../obj-x86_64-linux-gnu/generated/glib/c/functions.d(1187): Error: semicolon expected following function declaration
../obj-x86_64-linux-gnu/generated/glib/c/functions.d(1187): Error: declaration expected, not `*`
[2/327] ldc2 -Iglibd-2.0@sha -I. -I.. -Igenerated -I../generated -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -enable-color -O -g -release -wi -relocation-model=pic   -of 'glibd-2.0@sha/obj-x86_64-linux-gnu_generated_gtkd_paths.d.o' -c ../obj-x86_64-linux-gnu/generated/gtkd/paths.d
[3/327] ldc2 -Iglibd-2.0@sha -I. -I.. -Igenerated -I../generated -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -enable-color -O -g -release -wi -relocation-model=pic   -of 'glibd-2.0@sha/obj-x86_64-linux-gnu_generated_gtkd_Implement.d.o' -c ../obj-x86_64-linux-gnu/generated/gtkd/Implement.d
[4/327] ldc2 -Iglibd-2.0@sha -I. -I.. -Igenerated -I../generated -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -enable-color -O -g -release -wi -relocation-model=pic   -of 'glibd-2.0@sha/obj-x86_64-linux-gnu_generated_gtkd_Loader.d.o' -c ../obj-x86_64-linux-gnu/generated/gtkd/Loader.d
ninja: build stopped: subcommand failed.
```
See https://buildd.debian.org/status/fetch.php?pkg=glib-d&arch=amd64&ver=2.0.0-1%2Bb1&stamp=1536513253&raw=0 for a full build log.
I assume this is actually an issue in gir-to-d, that's why I am reporting it here (but it could also be a problem in GLibD's wrap files, I didn't have the time yet to reproduce the issue locally).

See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=908671 for the respective bug report at Debian.

**Comment by MikeWey** [comment]
The prototype of a new function in GLib 2.58 wasn't generated properly.

Should be fixed in 0.16.1.

**Comment by ximion** [comment]
Thank you for the quick fix!
